www/src/components/prev-and-next.js

```diff
@@ -0,0 +1,92 @@
+import React from "react"
+import { Link } from "gatsby"
+import ArrowForwardIcon from "react-icons/lib/md/arrow-forward"
+import ArrowBackIcon from "react-icons/lib/md/arrow-back"
+
+import {
+  colors,
+  space,
+  mediaQueries,
+  lineHeights,
+  fontSizes,
+  fonts,
+} from "../utils/presets"
+
+const prevNextLinkStyles = {
+  "&&": {
+    borderBottom: 0,
+    color: colors.gatsby,
+    fontFamily: fonts.header,
+    fontSize: fontSizes[3],
+    fontWeight: `bold`,
+    lineHeight: lineHeights.dense,
+  },
+}
+const prevNextLabelStyles = {
+  color: colors.text.secondary,
+  fontSize: fontSizes[2],
+  fontWeight: `normal`,
+  marginBottom: space[2],
+  marginTop: 0,
+}
+
+const PrevAndNext = ({ prev = null, next = null, ...props }) => {
+  if (!prev && !next) {
+    return null
+  }
+
+  return (
+    <div
```

Comment:
It would help to mark these as navigation - I think both of the links could be wrapped in `<nav aria-label="pagination">` or similar